json/flags.json

```
@@ -49,6 +49,20 @@
           "$ref": "./targeting.json"
         }
       }
+    },
+    "metadata": {
+      "title": "Flag Set Metadata",
+      "description": "Metadata about the flag set, with keys of type string, and values of type boolean, string, or number.",
+      "properties": {
+        "flagSetId": {
```

```suggestion
        "id": {
```

I think the rest is implicit